intermediate_notebooks/E2E/taxi/NYCTaxi-E2E-GPU-Dask.ipynb

```diff
@@ -0,0 +1,1643 @@
+      "text/plain": [
+       "<Figure size 1080x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = X_test_pd['trip_distance'].hist(bins=30, figsize=(15,5), color='red')\n",
+    "ax.set_yscale('log')\n",
+    "ax.set_xlabel(\"trip distance (miles)\")\n",
+    "ax.set_ylabel(\"count\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## VISUALS WITH CUXFILTER"
```

Comment:
Would this make more sense in the EDA section? IMO it would make the notebook flow more naturally if so, rather than seemingly having an appendix of visualizations at the end.

Reply:
I can add that before the XGB model. This is to visualize data after cleaning data and applying queries.